app/views/gitlab_oauth.py

```diff
@@ -38,6 +41,7 @@ def gitlab_authenticate(request):
         )
 
     if user is None:
+        logging.error(f"{gitlab_user_name}'s account not created.")
```

Comment:
Can also be rephrased as - 
Github account for {gitlab_user_name} was not created.